module/documents/item.mjs

```diff
@@ -1250,7 +1237,9 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     );
 
     let item = this;
-    if ( spellLevel && (this.type === "spell") ) item = item.clone({ "system.item.level": spellLevel });
+    if ( spellLevel && (this.type === "spell") ) {
+      item = item.clone({ "flags.dnd5e.scaling": Math.max(0, item.system.level - spellLevel) }, { keepId: true });
```

Comment:
Has `item.system.level` already been increased here, and `spellLevel` is the base level?

Reply:
Nope, got the math backwards :O